Library/Homebrew/cmd/home.rb

```diff
@@ -5,27 +5,29 @@
 require "cask/exceptions"
 
 module Homebrew
+  include Kernel
```

Comment:
Not including this line causes Sorbet to report error [7003](https://sorbet.org/docs/error-reference#7003) for the `exec_browser` and `puts` methods.

Reply:
Hmm, again this seems like weird behaviour from Sorbet. Does an explicit `require` fix it? Kernel is already included implicitly.

Reply:
`require` does not help here. :/

Reply:
The behaviour is because it tries to consider how modules can be mixins.

Basically the inheritance goes:

```
Class/Module -> Object -> BasicObject
```

Ruby by default has `include Kernel` in `Object` so anything that inherits from `Object` (which classes and modules do by default) will be able to use everything in `Kernel`.

However Sorbet's trying to think of scenarios where one could do this:

```
class Example < BasicObject
  include Homebrew
end
```

Now this is obviously something we'll never do but that's where Sorbet is coming from.

This is a complete guess but maybe putting something like:

```
module Homebrew
  include Kernel
end
```

at the end of `utils.rb` or wherever the best place is. If having the `include Kernel` in one place is possible (I haven't tested), it would be good idea to do so.

Reply:
Yeh, this came up in Slack, thanks 👍. I reckon in `global.rb` (if that's enough, hopefully it is).